lib/navigation.php

```diff
@@ -277,11 +277,24 @@ function gutenberg_migrate_menu_to_navigation_post( $new_name, $new_theme, $old_
 	}
 	remove_filter( 'option_stylesheet', $get_old_theme_stylesheet );
 
-	update_option( 'fse_navigation_areas', $area_mapping );
+	update_option( 'wp_navigation_areas', $area_mapping );
 }
 
 add_action( 'switch_theme', 'gutenberg_migrate_menu_to_navigation_post', 200, 3 );
 
+/**
+ * Retrieves navigation areas.
+ *
+ * @return array Navigation areas.
+ */
+function gutenberg_get_navigation_areas_menus() {
+	$areas = get_option( 'wp_navigation_areas', array() );
+	if ( ! $areas ) {
+		$areas = get_option( 'fse_navigation_areas', array() );
```

Comment:
I think we should make `fse_navigation_areas` a variable `$legacy_key_name` and then add a comment to explain why we're falling back to this option.